### ISSUE TITLE
`cl run` with wait option throws error

### ISSUE THREAD

**Issue by kelvinguu** [codalab/codalab-worksheets#287]
`cl run -v 'echo hello'`

``` python
Traceback (most recent call last):
  File "/Users/Kelvin/Dropbox/projects/codalab-cli/codalab/bin/../../codalab/bin/cl.py", line 66, in <module>
    run_cli()
  File "/Users/Kelvin/Dropbox/projects/codalab-cli/codalab/bin/../../codalab/bin/cl.py", line 57, in run_cli
    cli.do_command(sys.argv[1:])
  File "/Users/Kelvin/Dropbox/projects/codalab-cli/codalab/lib/bundle_cli.py", line 644, in do_command
    structured_result = command_fn()
  File "/Users/Kelvin/Dropbox/projects/codalab-cli/codalab/lib/bundle_cli.py", line 626, in <lambda>
    command_fn = lambda: args.function(self, args)
  File "/Users/Kelvin/Dropbox/projects/codalab-cli/codalab/lib/bundle_cli.py", line 1004, in do_run_command
    self.wait(client, args, uuid)
  File "/Users/Kelvin/Dropbox/projects/codalab-cli/codalab/lib/bundle_cli.py", line 986, in wait
    self.do_info_command([uuid, '--verbose'], self.create_parser('info'))
AttributeError: 'BundleCLI' object has no attribute 'create_parser'
```

Looks like BundleCLI used to have a method called `create_parser`, but not anymore?

**Comment by percyliang** [comment]
Seems to have broken in the overhaul of the CLI parser...thanks for pointing it out.

**Comment by a10y** [comment]
@kelvinguu I'm also on OSX and I can't reproduce this on latest master/develop, out of curiosity what revision are you on?

**Comment by kelvinguu** [comment]
Hi Andrew, I'm seeing this on the latest revision of develop: 0308998.
Thanks!

**Comment by kashizui** [comment]
I found the problem—it has to do with the `-W` wait flag, not the verbose flag. There were a couple things not picked up in the CLI refactor, I'll take care of this soon.

**Comment by kashizui** [comment]
Fixed by #318
